server/src/main/kotlin/io/spine/examples/pingh/server/datastore/TestDatastoreStorageFactory.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.pingh.server.datastore
+
+import com.google.cloud.datastore.Datastore
+import io.spine.server.storage.datastore.DatastoreStorageFactory
+import io.spine.server.storage.datastore.DatastoreWrapper
+import io.spine.server.storage.datastore.DsColumnMapping
+import io.spine.server.storage.datastore.ProjectId
+
+/**
+ * A test implementation of the [DatastoreStorageFactory].
+ *
+ * @param datastore The `Datastore` for working with data.
+ */
+public class TestDatastoreStorageFactory private constructor(datastore: Datastore) :
```

Comment:
Please consider using Spine-provided `testutil` artifact. See [this](https://github.com/SpineEventEngine/gcloud-java/tree/v1/testutil-gcloud/src/main/java/io/spine/testing/server/storage/datastore).